style.css

```diff
@@ -0,0 +1,806 @@
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 6vh;
+  margin-left: 3vw;
+  margin-right: 3vw;
+  margin-top: 20vh;
+}
+
+.feature-text {
+  color: gray;
+  line-height: 1.5rem;
+  margin-left: 6vw;
+  margin-right: 6vw;
+  padding-bottom: 3vh;
+  text-align: center;
+}
+
+.feature-title {
+  font-size: 26px;
+  margin-bottom: 1vh;
+}
+
+/* Footer ***********************************************************/
+.footer-container {
+  align-items: center;
+  background: rgb(135, 135, 216);
+  color: white;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 3vh;
+  padding-top: 3vh;
+  text-align: center;
+}
+
+.footer-item {
+  padding-bottom: 4vh;
+}
+
+.footer-logo {
+  padding-bottom: 6vh;
+}
+
+.footer-logo {
+  fill: white;
+  stroke: white;
+}
+.footer-logo-img {
+  /* filter: brightness(0) invert(1); */
+  /* filter: invert(90%) sepia(23%) saturate(6245%) hue-rotate(163deg) brightness(97%) contrast(101%); */
+}
+
+.footer-img-two {
+  padding-left: 2vw;
+}
+
+/* Info ***********************************************************/
+.info-btn {
+  background: indianred;
+  border: none;
+  border-radius: 4px;
+  color: white;
+  padding-bottom: 2vh;
+  padding-top: 2vh;
+}
+
+.info-container {
+  background: hsl(230, 70%, 62%);
+  color: white;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 10vh;
+  padding-top: 10vh;
+  text-align: center;
+}
+
+.info-input {
+  border: none;
+  border-radius: 4px;
+  margin-bottom: 3vh;
+  padding-bottom: 2vh;
+  padding-left: 2vw;
+  padding-top: 2vh;
+  width: 240px;
+}
+
+.info-input-button-wrapper {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  padding-top: 3vh;
+  width: 70%;
+}
+
+.info-text-one {
+  padding-bottom: 2vh;
+}
+
+.info-text-two {
+  font-size: 26px;
+  padding-bottom: 2vh;
+}
+
+/* Navbar ***********************************************************/
+#hamburgerMenu {
+  display: inline-block;
+}
+
+.nav-item {
+  display: none; 
+} 
+
+.nav-login-btn {
+  display: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-bottom: 8vh;
+  padding-left: 3vw;
+  padding-right: 5vw;
+  padding-top: 6vh;
+}
+
+.nav-right {
+  display: none;
+}
+
+.nav-right-mobile {
+  overflow: hidden;
+  position: relative;
+}
+
+.nav-right-mobile a {
+  display: none;
+}
+
+/* .nav-right-mobile a {
+
+} */
+
+.nav-right-mobile a.icon {
+  background: white;
+  color: black;
+  display: block;
+  left: 0;
+  position: absolute;
+  right: 0;
+}
+
+
+
+
+/* MEDIA QUERIES ********************************************************/
+/************************************************************************/
+@media(min-width: 768px) {
+/* Bookmark ***********************************************************/
+.bookmark-background {
+  background: hsl(230, 70%, 62%);
+  border-radius: 70% 0% 0% 70%;
+  margin-left: 14vw;
+  margin-top: 14vh;
+  height: 35vh;
+  width: 25vw;
```

Comment:
Maybe change the bookmark-change to something like this:    
border-radius: 44% 0% 0% 44%;
width: 41vw;